examples/canvas_more/pyproject.toml

```diff
@@ -0,0 +1,44 @@
+[build-system]
+requires = ["briefcase"]
+
+[tool.briefcase]
+project_name = "Canvas Demo 2"
+bundle = "org.beeware"
+version = "0.0.1"
+url = "https://beeware.org"
+license = "BSD license"
+author = 'Barak Korren'
+author_email = "bkorren@redhat.com"
```

Comment:
This field is intended as a "who do I speak to when things go wrong" contact, rather than an "original author" attribution. We've been using `Tiberius Yak` as the author, `tiberius@beeware.org` as the email. 

I don't want to erase the significance of your contribution, but I'd rather keep the "generic" contact details. If only to protect your own inbox in years to come should you choose to end your association with the project. Code attribution will always live on in the git history.

Reply:
Sure, will change n/p. I wans't sure this is the right thing to put here anyway